decidim-proposals/spec/system/admin/filter_proposals_spec.rb

```diff
@@ -6,7 +6,7 @@
   include_context "when admin manages proposals"
   include_context "with filterable context"
 
-  STATES = Decidim::Proposals::Proposal::POSSIBLE_STATES.map(&:to_sym)
+  STATES = Decidim::Proposals::Proposal::STATES.keys.map(&:to_sym)
```

Comment:
Isn't it already a symbol? I think we can remove `.to_sym`.
```suggestion
  STATES = Decidim::Proposals::Proposal::STATES.keys
```